packages/xo-web/src/xo-app/settings/plugins/index.js

```diff
@@ -251,29 +255,56 @@ class Plugin extends Component {
   }
 }
 
-@addSubscriptions({
-  plugins: subscribePlugins,
-})
-export default class Plugins extends Component {
-  render() {
-    if (isEmpty(this.props.plugins)) {
-      return (
-        <p>
-          <em>{_('noPlugins')}</em>
-        </p>
-      )
-    }
+const compareNames = (a, b) => (a.name < b.name ? -1 : 1)
 
-    return (
+export default decorate([
+  addSubscriptions({
+    plugins: subscribePlugins,
+  }),
+  provideState({
+    initialState: () => ({ filter: '' }),
+    effects: { linkState },
+    computed: {
+      filteredPlugins: ({ predicate }, { plugins }) =>
+        predicate === undefined ? plugins : plugins.filter(predicate),
+      predicate: ({ filter }) => {
+        if (filter.trim() === '') {
+          return
+        }
+
+        try {
+          return ComplexMatcher.parse(filter).createPredicate()
+        } catch (error) {
+          console.warn(error)
+        }
+      },
+      sortedPlugins: ({ filteredPlugins }) =>
+        filteredPlugins.sort(compareNames),
```

Comment:
`_.sortBy(filteredPlugins, 'name')`?

Reply:
I didn't need advanced features so I decided to go with the simplest (fastest) solution.

Would you like me to change it?

Reply:
It would allow you to remove `compareNames` so I'd say yes.